changelog/unreleased/kong/disallow_add_del_workspaces.yml

```diff
@@ -0,0 +1,4 @@
+message: |
```

Comment:
Also, let's not mention this in the changelog as this has never been a user-visible feature of OSS.